Google.Cloud.EntityFrameworkCore.Spanner/Storage/Internal/SpannerSqlGenerationHelper.cs

```diff
@@ -61,13 +61,15 @@ public override void DelimitIdentifier(StringBuilder builder, string name, strin
 
         /// <inheritdoc />
         public override string DelimitIdentifier(string identifier)
-            => $"{EscapeIdentifier(GaxPreconditions.CheckNotNullOrEmpty(identifier, nameof(identifier)))}";
+            => $"`{EscapeIdentifier(GaxPreconditions.CheckNotNullOrEmpty(identifier, nameof(identifier)))}`";
```

Comment:
We need to override `EscapeIdentifier` as well, as the default implementation of `EscapeIdentifier` will escape `"` (double quotes). That means that the following now happens:
1. `Some"Identifier` => `` `Some\"Identifier` ``
2. ``Some`Identifier`` => `` `Some`Identifier` ``

While it should be:
1. `Some"Identifier` => `` `Some"Identifier` ``
2. ``Some`Identifier`` => `` `Some\`Identifier` ``